app/models/veteran_enrollment_system/form1095_b/form1095_b.rb

```diff
@@ -0,0 +1,205 @@
+# frozen_string_literal: true
+
+require 'common/models/resource'
+
+module VeteranEnrollmentSystem
+  module Form1095B
+    class Form1095B
```

Comment:
Not gonna lie, I hate this naming convention, but Form1095B was already taken by the old version and I can't easily rename it because it's a database record. I could have created V1 folder and called it V1::Form1095B (this is what the mobile team does), but I didn't want to do that because it would be establishing a new pattern. This is the most inline with current models, and I believe there are already some models with duplicated names like this. It's an unfortunate side effect of our code ownership conventions. We could change this to Form1095B after we deprecate the old model, but I doubt we will.